clang/test/Analysis/Checkers/WebKit/retain-ptr-ctor-adopt-use.mm

```diff
@@ -97,3 +97,13 @@ void create_member_init() {
 RetainPtr<id> return_bridge_cast() {
   return bridge_cast<CFArrayRef, NSArray>(create_cf_array());
 }
+
+void mutable_copy() {
+  RetainPtr<NSMutableDictionary> mutableArray = adoptNS(@{
+    @"Content-Type": @"text/html",
+  }.mutableCopy);
+}
```

Comment:
This is a question for my understanding: Is the behavior expected to be same/different for `CFArrayCreateMutable`?

Reply:
Right but I didn't add a test for it separately since `CFArrayCreateMutable` contains `Create` in its name and it's a regular function call so the regular `Create` function rule applies there.

Reply:
Got it, thank you for explaining.